packages/client/test/TestServer.ts

```diff
@@ -0,0 +1,374 @@
+import * as fsv from './generated/flight/Flight.grpc-server'
+import * as grpc from '@grpc/grpc-js'
+import * as flt from '../src/generated/flight/Flight'
+import {MetadataVersion} from 'apache-arrow/fb/metadata-version'
+import {FieldNode, Message} from 'apache-arrow/ipc/metadata/message'
+import {MessageHeader} from 'apache-arrow/fb/message-header'
+import {Schema} from 'apache-arrow/Arrow.node'
+import {Log, setLogger, consoleLogger} from '../src'
+
+setLogger({
+  error(message: string, error) {
+    consoleLogger.error(message, error)
+  },
+  warn(message: string, error) {
+    consoleLogger.warn(message, error)
+  },
+})
+
+// todo - when run in parallel can get conflicts?
+// so may need to increment port number or isolate runs
```

Comment:
Fix this TODO or remove this comment from sources.

Reply:
I created a temporary script to check behavior in a parallel run and then ran it.  I also tried the same for  `test:unit`
```
"test:integration:parallel": "mocha 'test/integration/**/*.test.ts' --parallel --exit",
```
I did not see any negative side effects - for now.  In fact parallel runs were a second or two slower.  The project is small enough that test parallelization brings little added benefit.  

I've removed this `todo` comment.  

Still, I think there are some ways in which this TestServer can be improved.  However,  at the moment they are of a lower priority.